package.json

```diff
@@ -24,7 +24,8 @@
     "prettify": "balena-lint -e ts -e js --typescript --fix src/ test/ typings/ build-utils/ webpack.config.js",
     "release": "tsc --project tsconfig.release.json && mv build/src/* build",
     "sync": "ts-node --files sync/sync.ts",
-    "clean": "rimraf build"
+    "clean": "rimraf build",
+    "preinstall": "./install-dbus.sh || true"
```

Comment:
One last thought. Maybe the script could go in `build-utils`?

Reply:
Good shout. https://github.com/balena-os/balena-supervisor/commit/b680d4996657346a988cd8c45ac7f12a234a1662